x/bank/keeper/send.go

```diff
@@ -325,24 +336,26 @@ func (k BaseSendKeeper) setBalance(ctx sdk.Context, addr sdk.AccAddress, balance
 	}
 
 	accountStore := k.getAccountStore(ctx, addr)
+	newAccStore := store2.NewStoreAPI(accountStore)
```

Comment:
nit: these names are a bit confusing

Reply:
I know, but I don't really get what should I name these into. Any suggestions ?

Reply:
Would be too bad if we do something like this?:
```
	accountStore := store.NewStoreAPI(k.getAccountStore(ctx, addr))
```

Reply:
> Would be too bad if we do something like this?:
> 
> ```
> 	accountStore := store.NewStoreAPI(k.getAccountStore(ctx, addr))
> ```

not at all. Better than previous lines.
stupid me, haven't thought that :/